velox/docs/functions/spark/datetime.rst

```diff
@@ -93,6 +93,10 @@ These functions support TIMESTAMP and DATE input types.
         SELECT from_unixtime(3600, 'yyyy'); -- '1970'
         SELECT from_unixtime(9223372036854775807, "yyyy-MM-dd HH:mm:ss");  -- '1969-12-31 23:59:59'
 
+    If we run the following query in the `Asia/Shanghai` time zone: ::
```

Comment:
Would it be possible to include all 3 queries above, but in this time zone?

```
The same queries evaluated with `Asia/Shanghai` session time zone return timestamps shifted by 8 hours.

SELECT ...
```

Reply:
Sure. See 15e85ed4049f0993047e4147c647fe0d50a4cba0